AmplitudeServiceKit/en.lproj/Localizable.strings

```diff
@@ -0,0 +1,2 @@
+/* The title of the Amplitude service */
+"Amplitude" = "Amplitude";
```

Comment:
Maybe I'm mistaken, but won't the default language localization be provided when a `.strings` file is missing. If yes, I don't see the point of making `.strings` files that do not provide any localization.

Reply:
Yes, good call. Will remove in the three service repos.